static/dtale/DataViewer.jsx

```diff
@@ -238,7 +238,7 @@ class ReactDataViewer extends React.Component {
                       onScroll={this.props.closeColumnMenu}
                       cellRenderer={this._cellRenderer}
                       height={gridHeight}
-                      width={width - 3}
+                      width={width - (this.props.menuPinned ? 185 : 3)}
```

Comment:
Need to account for the scrollbar 3px when its pinned as well. I'll add this update
```suggestion
                      width={width - (this.props.menuPinned ? 188 : 3)}
```